app/views/companies/show.html.haml

```diff
@@ -1,51 +1,49 @@
-%header.entry-header
-  %h1.entry-title= @company.name
-  .post-title-divider
+%div{ :id => "#{@company.class.name.downcase}-#{@company.id}", class: "show-item #{@company.class.name.downcase} #{@company.class.name.downcase}-#{@company.id}" }
```

Comment:
No, what I meant was why bother with getting the class of the instance var, etc .... as opposed to
```
%div{ :id => "company-#{@company.id}", class: "show-item company company-#{@company.id}" }
```